docs/source/contributing/developing.adoc

```diff
@@ -360,6 +360,15 @@ This information is often enough to find and debug the reason for a failure.
 However, for cases which require further investigation, the integration test also logs more detailed progress into a log file.
 This file is located at *_$GOPATH/github.com/minishift/minishift/out/test-results/integration_YYYY-MM-DD_HH:MM:SS.log_*.
 
+
+[[integration-tests-as-package]]
+==== Using Integration Tests as a Package
+
+Integration tests can be also used as a package outside of the {project} repository.
+You can import it as `github.com/minishift/minishift/test/integration/testsuite`.
+As a starting point for your project you can copy and use the `test/integration/integration_test.go` file.
```

Comment:
Hyperlink `test/integration/integration_test.go` to respective file is needed.

Reply:
@budhrg thanks for suggestion, should be fixed now: `link:https://github.com/minishift/minishift/blob/master/test/integration/integration_test.go[`test/integration/integration_test.go`]`